### ISSUE TITLE
Please add -v --verbose argument

### ISSUE THREAD

**Issue by kuban33** [tuxera/ntfs-3g#71]
Hello ntfs-3g Team,
please could You add standard -v argument for verbose mode for scripts debug level logging.

Thank You.

**Comment by unsound** [comment]
Hi, what specifically do you want logged that isn't logged today?

**Comment by kuban33** [comment]
Hello, example of verbose output -v would be:
`ntfs-3g -v /dev/da0p1 /media/exthdd0`
`/dev/fuse on /media/exthdd0 (fusefs, fsid 0000000000000000)`

**Comment by unsound** [comment]
I'm not sure what information this provides that you cannot get elsewhere. E.g. from the ```mount``` or ```mountpoint``` commands. I think you need to describe your use case better... what problem are you trying to solve?

**Comment by kuban33** [comment]
Yes, You are right, You can definitely get this info from `mount` command.
I think it would be useful and straightforward for logging when You create shell scripts like standard BSD mount command have this -v option.
This is definitely not an issue more kind of nice to have request for further versions.
Thanks.

**Comment by unsound** [comment]
I see, now it's more clear what the request is trying to accomplish. The ```mount``` command outputs different info when the ```-v``` option is specified for different systems, e.g. Linux has a different output format than FreeBSD and it's even localized. It seems difficult to replicate this output accurately for all systems so I don't think we'll implement this, but I'll keep this open with low priority and see if there is more interest out there for this feature.
